src/notify.js

```diff
@@ -35,7 +36,6 @@ class Toast extends React.Component {
 
 		const containerStyle = {
 			position: 'fixed',
-			width: '50%',
```

Comment:
I actually removed this as it didn't have an effect on the width. The `0px` left and right on a fixed position element would override the width settings.